<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmHost</class>
 <widget class="QDialog" name="frmHost">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>主机管理</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/image/main.png</normaloff>:/image/main.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;宋体&quot;;</string>
  </property>
  <widget class="QListWidget" name="listHost">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>391</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="labHostCodeCF">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>算法:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labHostID">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>通道号:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="typeBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>14</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>IPC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FILE</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>54</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
    <property name="icon">
     <iconset resource="main.qrc">
      <normaloff>:/image/delete.png</normaloff>:/image/delete.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtHostCodeBF">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>94</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>3C C3 01 00 16 3E 01 56 01 09 01 3E 01 57 01 08 00 3E 01 50 01 08 01 3C 02 52 D7 3C 0D 0A</string>
    </property>
    <property name="cursorPosition">
     <number>89</number>
    </property>
   </widget>
   <widget class="QLabel" name="labHostType">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>主机类型:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addressLine">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>94</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>rtsp:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labHostCOM">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>地址:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>94</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
    <property name="icon">
     <iconset resource="main.qrc">
      <normaloff>:/image/excel.png</normaloff>:/image/excel.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>14</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
    <property name="icon">
     <iconset resource="main.qrc">
      <normaloff>:/image/add.png</normaloff>:/image/add.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="algoBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>无</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>安全帽识别</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>行人车辆识别</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>人脸识别</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>口罩识别</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>抽烟识别</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>车牌识别</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="channelBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="waitAlgoNumLine">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="labHostCodeCF_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>算法执行间隔：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labHostCodeCF_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>帧</string>
    </property>
   </widget>
   <widget class="QLabel" name="labHostCodeCF_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>帧间隔时间：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="waitFrameLine">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
   <widget class="QLabel" name="labHostCodeCF_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ms</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>listHost</tabstop>
  <tabstop>typeBox</tabstop>
  <tabstop>addressLine</tabstop>
  <tabstop>txtHostCodeBF</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>btnSave</tabstop>
 </tabstops>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
